src/components/MyAccount/CheckoutsTab.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  Box,
+  Button,
+  Icon,
+  Link,
+  Table,
+  Text,
+  useModal,
+} from "@nypl/design-system-react-components"
+import type { Checkout, Patron } from "../../types/accountTypes"
+import { useEffect, useState } from "react"
+import { BASE_URL } from "../../config/constants"
+import { getTitle } from "../../utils/myAccountUtils"
+import styles from "../../../styles/components/MyAccount.module.scss"
+
+const CheckoutsTab = ({
+  checkouts,
+  patron,
+}: {
+  checkouts: Checkout[]
+  patron: Patron
+}) => {
+  const checkoutsHeaders = [
+    "Title",
+    "Barcode",
+    "Call number",
+    "Due back by",
+    "Manage checkout",
+  ]
+
+  function RenewButton(checkout, patron) {
```

Comment:
This component is big enough that it should be defined outside of `CheckoutsTab`. Keeping it in this file is fine or consider moving it to a new file.